ext/standard/metaphone.c

```diff
@@ -204,24 +209,26 @@ static void metaphone(unsigned char *word, size_t word_len, zend_long max_phonem
 	case 'G':
 	case 'K':
 	case 'P':
-		if (Next_Letter == 'N') {
+		if (Read_Next_Letter == 'N') {
 			Phonize('N');
 			w_idx += 2;
 		}
 		break;
 		/* WH becomes W,
 		   WR becomes R
 		   W if followed by a vowel */
-	case 'W':
-		if (Next_Letter == 'R') {
-			Phonize(Next_Letter);
+	case 'W': {
+		char next_letter = Read_Next_Letter;
+		if (next_letter == 'R') {
+			Phonize(next_letter);
```

Comment:
Cannot this be replaced by a constant operation?
```suggestion
		if (next_letter == 'R') {
			Phonize('R');
```
Not that it seems to matter, as the ``Phonize`` macro seems to extend a string sometimes...

Reply:
Sure, I'll update it